### DISCUSSION TITLE
Top bar, numbers just near the keyboard icon

### DISCUSSION THREAD

**Discussion by Jean-Gabriel1982** [GrandOrgue/grandorgue#1346] [General]
Hello,
In my configuration, this number (just near the Panic button) was put on 5.800. 

For tests I've changed to 9.000, save and quit. 
Now if I want to come back to 5.800, I put this number, save and quit. 

Restarting GO, the number is still 9.000

**Comment by midimusic** [comment]
This number is "polyphony" and it has a default value of 2900. I would think that this would be enough for any normal organ. Why do you think you need to change it?

I can confirm that this polyphony number IS saved by "File/Save" followed by "File/Exit" so I think you must be mistaken. 

csw900

**Comment by thbeckpr** [comment]
Polyphony numbers higher than the default value are recommended by Sonus Paradisi for many of their sample sets. The recommended polyphony for the William Hill English sample set is 6000.

I'm not sure what you mean by a "normal" organ. Maybe that means any organ that you play. Certainly there is a wide range of sample sets available. If there were no need to change the default polyphony then there would not be an option.

And you also do not know that you are using the same version of GrandOrgue so telling someone that they "must be mistaken" is not a helpful answer.

**Reply by midimusic** [reply]
You have only 10 fingers and two feet. A polyphony of 6000 means that you can play 6000/12  = 500 notes simultaneously for each finger and foot. Even allowing for couplers and extended releases I think this would be quite a difficult thing to do and certainly not "normal". 

csw900

**Comment by jason-barba** [comment]
I find https://www.digitalpianoreviewguide.com/how-much-polyphony-do-i-need-for-my-piano/ interesting. 
Three stops and 3 notes at the same time requires >= 9 polyphony.

**Reply by midimusic** [reply]
An interesting article. GO appears to have "first note priority", i.e. once the maximum polyphony is reached additional notes do not play. Also these additional notes do not play when the first notes are released (unless they are re-played).

As an experiment - you can try reducing GO polyphony to 10 and find out what really happens when polyphony is exceeded.

csw900

**Comment by Jean-Gabriel1982** [comment]
> This number is "polyphony" and it has a default value of 2900. I would think that this would be enough for any normal organ. Why do you think you need to change it?
> 
> 
> 
> I can confirm that this polyphony number IS saved by "File/Save" followed by "File/Exit" so I think you must be mistaken. 
> 
> 
> 
> csw900
> 
> 

Hello,
I just made as you said but the number is still 9000 ...

**Reply by midimusic** [reply]
What is your operating system and what is your GO version? I am using windows and GO v3.2.0-1 and do not have your problem. You may have found a bug. I refer this to Oleg for a solution - but in the meantime reinstalling GO may cure your problem or you can just ignore it. GO will almost certainly work just as well whatever this number is - provided it is above the default.
csw900

**Reply by Jean-Gabriel1982** [reply]
I use GO 3.9.1-1 and Linux Mint 21 Vanessa base Ubuntu 22.04

**Reply by midimusic** [reply]
Is this the ONLY setting that is not saved? It is possible that this problem could be caused by a write protected folder.

It would be useful if someone else with these versions can confirm the existance of this potential bug.

csw900.

**Comment by thbeckpr** [comment]
There is definitely a bug. 

I'm using GO v3.7.1-1 on xubuntu 22.04.1

This doesn't save the polyphony change:
load GO
change polyphony 
quit GO
on reload, no change

This also doesn't save the polyphony change:
load sample set
change polyphony
change setting
save
quit

This saves the polyphony change:
load GO
change polyphony
load sample set
save
quit

This also saves the polyphony change:
load GO
change polyphony 
load sample set
change a setting
save
quit

So it seems that unless the polyphony setting change is made first, it won't be saved.

The only setting change that I made was a pipe amplitude. I didn't try changing other settings.
 
Also the state of the Save menu option is enabled after loading a sample set even when no changes have been made.

**Reply by rousseldenis** [reply]
@thbeckpr Could you try with the last version ?

**Reply by thbeckpr** [reply]
With v3.9.4-1 the result is the same.
